ert_gui/tools/plot/plot_api.py

```diff
@@ -0,0 +1,124 @@
+from ert_data import loader as loader
+import pandas as pd
+
+
+class PlotApi:
```

Comment:
This class should be tested as it's central to the GUI and potentially the future data api.